features/notificationPermissions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Files are named using - as separators (kebab case) so this file should be `noticiation-permissions.json`

Reply:
Ok, so just to make sure. This file name doesn't need to match the feature name on `android-override.json`, right?

Reply:
It does, otherwise it has no impact. The kebab is removed and camelcased to match the feature name.